### DISCUSSION TITLE
Project licence is not being applied for a new user

### DISCUSSION THREAD

**Discussion by amca01** [sagemathinc/cocalc#6341] [Q&A]
Hello,  I'm running a short course in Sage using CoCalc, and I have bought a 20 user licence for a month.  I thought I had created my project using this licence.  However, as an experiment, I logged in as a student using a different email address, and opened the project, to be told that I had logged in as a "free trial" and that without a licence I could expect very poor behaviour.

How can I be sure that my licence is applied to the project - is there any way I can "attach" a licence to a project?  (It consists of only two files: a jupyter notebook for experimenting, which I intend to distribute to all students, and a markdown file with some vague instructions.)  Given that I have created the files, I could easily start from scratch ... if that's the only way to do it.

Thanks, Alasdair

**Comment by williamstein** [comment]
>  is there any way I can "attach" a licence to a project? 

Yes, just open the project and go to the settings page for the project.  Everything about how that project is licensed is then visible and configurable there.
